packages/react-table/src/components/Table/examples/TableEditable.tsx

```diff
@@ -0,0 +1,296 @@
+            <TimesIcon />
+          </Button>
+        </div>
+      </Td>
+      <Td
+        dataLabel="Edit button"
+        className={css(inlineEditStyles.inlineEditAction, inlineEditStyles.modifiers.enableEditable)}
+      >
+        <Button
+          ref={editButtonRef}
+          aria-label={`Edit ${rowAriaLabel}`}
+          onClick={() => onClick('edit')}
+          onKeyDown={(event) => onKeyDown(event, 'edit')}
+          variant="plain"
+        >
+          <PencilAltIcon />
+        </Button>
+      </Td>
+    </>
+  );
+};
+
+interface EditableCellProps {
+  dataLabel: string;
+  staticValue: React.ReactNode;
+  editingValue: React.ReactNode;
+}
+
+const EditableCell: React.FunctionComponent<EditableCellProps> = ({ dataLabel, staticValue, editingValue }) => {
+  const hasMultipleInputs = Array.isArray(editingValue) && editingValue.every((elem) => React.isValidElement(elem));
+
+  return (
+    <Td dataLabel={dataLabel}>
+      <div className={css(inlineEditStyles.inlineEditValue)}>{staticValue}</div>
+      {hasMultipleInputs ? (
+        (editingValue as React.ReactElement[]).map((elem, index) => (
+          <div key={index} className={css(inlineEditStyles.inlineEditInput)}>
+            {elem}
+          </div>
+        ))
+      ) : (
+        <div className={css(inlineEditStyles.inlineEditInput)}>{editingValue}</div>
+      )}
+    </Td>
+  );
+};
+
+interface EditableRow {
+  data: CustomData;
+  columnNames: ColumnNames<CustomData>;
+  dataOptions?: CustomDataOptions;
+  saveChanges: (editedData: CustomData) => void;
+  ariaLabel: string;
+}
+
+const EditableRow: React.FunctionComponent<EditableRow> = ({
+  data,
+  columnNames,
+  dataOptions,
+  saveChanges,
+  ariaLabel
+}) => {
+  const [editable, setEditable] = React.useState(false);
+  const [editedData, setEditedData] = React.useState(data);
+
+  const inputRef = React.useRef();
+
+  return (
+    <Tr className={css(inlineEditStyles.inlineEdit, editable ? inlineEditStyles.modifiers.inlineEditable : '')}>
+      <EditableCell
+        dataLabel={columnNames.textInput}
+        staticValue={data.textInput}
+        editingValue={
+          <TextInput
+            aria-label={`${columnNames.textInput} ${ariaLabel}`}
+            ref={inputRef}
+            value={editedData.textInput}
+            onChange={(e) => setEditedData((data) => ({ ...data, textInput: (e.target as HTMLInputElement).value }))}
+          />
+        }
+      />
+      <EditableCell
+        dataLabel={columnNames.textInputDisabled}
+        staticValue={data.textInputDisabled}
+        editingValue={
+          <TextInput
+            aria-label={`${columnNames.textInputDisabled} ${ariaLabel}`}
+            isDisabled={true}
+            value={editedData.textInputDisabled ?? ''}
+          />
+        }
+      />
+      <EditableCell
+        dataLabel={columnNames.checkboxes}
+        staticValue={data.checkboxes.join(', ')}
+        editingValue={dataOptions?.checkboxes.map((option) => {
```

Comment:
Only other thing is that the checkboxes and radios for each row should be wrapped in a `pf-v5-c-inline-edit__group pf-m-column` element, with the checkbox group having `role="group"` and radios having `role="radiogroup"`. Each should also have an aria-label to the group container. Core markup (core needs to be updated to add the role and aria label to the checkbox group):

![image](https://github.com/patternfly/patternfly-react/assets/70952936/9748be63-09b4-4d86-8049-89f28a1fd813)